### DISCUSSION TITLE
Getting to the Mesh home page

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5929] [General]
Hi,
After I perform a server reset and log in I need to navigate back to the home page.
Is there a way to login straight to the home page after reset?
Thanks.

**Comment by si458** [comment]
The webpage should reload itself, or you click the reconnect string that appears in the middle of the screen

**Reply by ghost** [reply]
HI didn't understand what you mean

**Reply by si458** [reply]
sorry mis-read ur question,
when the server restarts and you re-login, it will always take you back to the same page you was at originally.
its rememered because of the `viewmode` thats set in the web URL
